pkg/kubelet/dockershim/convert.go

```diff
@@ -169,3 +170,18 @@ func toRuntimeAPISandbox(c *dockertypes.Container) (*runtimeapi.PodSandbox, erro
 		Annotations: annotations,
 	}, nil
 }
+
+func checkpointToRuntimeAPISandbox(id string, checkpoint *PodSandboxCheckpoint) *runtimeapi.PodSandbox {
+	state := runtimeapi.PodSandboxState_SANDBOX_NOTREADY
+	labels := make(map[string]string)
+	labels[kuberuntime.KubernetesManagedLabel] = "true"
```

Comment:
This is hacky...we'd be leaking the kubelet's internal implementation to the dockershim. I don't have a great solution to suggest right now, but would like to think about it more.